python/mage/geography/distance_calculator.py

```diff
@@ -55,3 +58,11 @@ def calculate_distance_between_points(
         distance *= KM_MULTIPLIER
 
     return distance
+
+
+class InvalidCoordinatesException(Exception):
```

Comment:
Could be useful to give the user the explanation of why coordinates are wrong invalid

Reply:
I did leave a message in the code above?